RedirectApplication/RedirectApplication/RedirectDB/RedirectRepository.cs

```diff
@@ -16,5 +17,11 @@ public Boolean IsRuleExist(String Url)
         {
             return context.RedirectRules.Any(x => x.TargetUrl == Url);
         }
+
+        public void AddRule(RedirectRule NewRule)
```

Comment:
Where this part of code is supposed to be used? Right now it looks like it is a dead code